magefiles/magefile.go

```diff
@@ -738,64 +744,51 @@ func AppendFrameworkDescribeGoFile(specFile string) error {
 
 }
 
-func newSprayProxy() (*SprayProxy, error) {
+func newSprayProxy() (*sprayproxy.SprayProxyConfig, error) {
 	sprayProxyUrl := os.Getenv("QE_SPRAYPROXY_HOST")
 	sprayProxyToken := os.Getenv("QE_SPRAYPROXY_TOKEN")
 	if sprayProxyUrl == "" || sprayProxyToken == "" {
 		return nil, fmt.Errorf("env var QE_SPRAYPROXY_HOST is not set")
```

Comment:
the error should report env `QE_SPRAYPROXY_HOST` or `QE_SPRAYPROXY_TOKEN` is not set